### DISCUSSION TITLE
solved! (I think) Pandoc-binary 1.1

### DISCUSSION THREAD

**Discussion by thekit** [imartinez/privateGPT#227] [General]
Hi Everyone,

I am trying to get started with PrivateGPT after seeing it in this youtube video by Prompt Engineering

https://www.youtube.com/watch?v=G7iLllmx4qc

as per the install notes of PrivateGPT on this github I grabbed the old Python 3.10 from the Microsoft App Store for Windows 11 home and then ran the pip installer command

everything went well until the last requirement pandoc-binary 1.1 which said 

`Collecting pandoc==2.3 (from -r requirements.txt (line 11))
  Using cached pandoc-2.3.tar.gz (33 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
ERROR: Could not find a version that satisfies the requirement pandoc-binary==1.11 (from versions: none)
ERROR: No matching distribution found for pandoc-binary==1.11`

I tried going to the Python 3.11 package but got the same error. Also I tried getting Pandoc from the official pandoc.org website and they only had Pandoc 3.1.2 with no information on what the binary package version is. 

in the end I went to 
https://pypi.org/project/pypandoc-binary/

and they said to run

`pip install pypandoc_binary`

which got me the version I wanted. really strange

`ERROR: Could not find a version that satisfies the requirement pandoc-binary==1.11 (from versions: none)
ERROR: No matching distribution found for pandoc-binary==1.11
PS C:\ai_experiments\privateGPT> pip install pypandoc_binary
Collecting pypandoc_binary
  Downloading pypandoc_binary-1.11-py3-none-win_amd64.whl (24.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 24.8/24.8 MB 11.5 MB/s eta 0:00:00
Installing collected packages: pypandoc_binary
Successfully installed pypandoc_binary-1.11`

I suspect the requirements.txt file is failing because it asks for `pandoc-binary` when it should have an underscore in the name like this `pypandoc_binary`

hope that helps!

**Comment by wally-kroeker** [comment]
yep that was it for me.  Thanks

**Reply by thekit** [reply]
I had to track down a few more dependencies which I logged in this discussion here https://github.com/imartinez/privateGPT/discussions/233
